task/06-conditions-n-loops-tasks.js

```diff
@@ -251,7 +331,20 @@ function reverseInteger(num) {
  *   4916123456789012 => false
  */
 function isCreditCardNumber(ccn) {
-    throw new Error('Not implemented');
+    //throw new Error('Not implemented');
+    let str = ccn.toString();
+    let sum = Number(str[str.length -1]);
+
+	for (let i = 0; i < str.length-1; i++) {
+        let digit = Number(str[i])
+        if( (i % 2) === (str.length % 2) ) {
```

Comment:
It is necessary to remove extra spaces in brackets and fix indentation after conditional statements
https://javascript.info/coding-style